chess-AI/experimental-work/state_representation.py

```diff
@@ -0,0 +1,138 @@
+import torch
+import chess
+import numpy as np
+
+class chess_state():

```

Comment:
Class name should be `ChessState`, see [naming in style guide](https://google.github.io/styleguide/pyguide.html#316-naming)